local-testing-docs/README.md

```diff
@@ -849,3 +850,15 @@ Please note that there will be unaware limitations, in this case please feel fre
   - Error experience
 
 > Snowflake employees can check [this engineering doc](https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/2792784931/Snowpark+Local+Testing+Development+Milestone) to see the open tasks and scheduling.
+
+
+# Telemetry
+
+By default, telemetry is turned on to collect the api usage information of unimplemented features(the names of functions/methods/expressions/types) for future works prioritization.
+It is expected that the client will still try to connect to the network and send data to Snowflake.
```

Comment:
It is legit for user to not have network access in a unit test setup. In this case, do the test fail, or do we just print a warning about telemetry cannot be sent?